pkg/deployment/inventory.go

```diff
@@ -53,12 +53,12 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 	}
 
 	for nodeName, node := range instance.Spec.Nodes {
-		host := nodeSetGroup.AddHost(nodeName)
+		host := nodeSetGroup.AddHost(strings.Split(node.HostName, ".")[0])
```

Comment:
should we make Hostname required?
https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/api/v1beta1/common.go#L52-L53

Reply:
It's defaulted to nodeName in webhook[1].

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/api/v1beta1/openstackdataplanenodeset_webhook.go#L59